README.md

```diff
@@ -63,6 +63,12 @@ bin/cake migrations mark_migrated 20150417223600
 # Since Migrations 1.3.1, a new `all` special value for the version argumentwas added.
 # The following will mark all migrations found as migrated.
 bin/cake migrations mark_migrated all
+
+# The following option `--target` will try to mark the migrations from beginning to the given version (including it)
+bin/cake migrations mark_migrated 20150417223600 --target
+
+# You can use the option `--exclude` along with `--target` and it will try to mark the migrations from beginning until the given version (excluding it)
+bin/cake migrations mark_migrated 20150417223600 --target --exclude
```

Comment:
That is a good point.
However, when the ``mark_migrated`` command was designed, it was to only mark a bake snapshot as migrated.
But now that it is becoming a full featured command, I agree it should comply with the rest of the commands.
If we are going down that path, that will break compability with userland code using the command.
It also should be known that when a snapshot is baked, all migrations up to that new one will be marked as migrated, unless we provide a way to target only one migration (even though off the top of my head, this behavior should not cause trouble).